src/Entity/Adherent.php

```diff
@@ -611,6 +620,11 @@ public function getGender(): ?string
         return $this->gender;
     }
 
+    public function getCustomGender(): ?string
```

Comment:
what about a setter?

Reply:
Never used the propertie is set by DTO, there are few setter in class Adherent